runserver.py

```diff
@@ -7,4 +7,4 @@
 app.config['APPLICATION_ROOT'] = '/'
 
 if __name__ == '__main__':
-    app.run(debug=True, port=8100)
+    app.run(debug=True, host='0.0.0.0', port=8100)
```

Comment:
In docker it would be much preferred to use a proper server and not the test one provided by flask/Werkzeug. Maybe in a different PR?

Reply:
@cthoyt, for now, this is a prototype to test some other aspects (not really meant for production).

In any case, would you care to give an example just to understand better what you mean?

Reply:
@nunogit I actually wrote up a bit on this the other day, might be interesting for you: https://cthoyt.com/2021/01/11/click-and-flask.html#using-gunicorn